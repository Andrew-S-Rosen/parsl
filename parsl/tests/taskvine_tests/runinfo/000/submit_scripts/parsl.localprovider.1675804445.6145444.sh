
export JOBNAME=$parsl.localprovider.1675804445.6145444
set -e
export CORES=$(getconf _NPROCESSORS_ONLN)
[[ "1" == "1" ]] && echo "Found cores : $CORES"
WORKERCOUNT=1
FAILONANY=0
PIDS=""

CMD() {
PARSL_WORKER_BLOCK_ID=0 vine_worker cclws21.cse.nd.edu 50055
}
for COUNT in $(seq 1 1 $WORKERCOUNT); do
    [[ "1" == "1" ]] && echo "Launching worker: $COUNT"
    CMD $COUNT &
    PIDS="$PIDS $!"
done

ALLFAILED=1
ANYFAILED=0
for PID in $PIDS ; do
    wait $PID
    if [ "$?" != "0" ]; then
        ANYFAILED=1
    else
        ALLFAILED=0
    fi
done

[[ "1" == "1" ]] && echo "All workers done"
if [ "$FAILONANY" == "1" ]; then
    exit $ANYFAILED
else
    exit $ALLFAILED
fi
