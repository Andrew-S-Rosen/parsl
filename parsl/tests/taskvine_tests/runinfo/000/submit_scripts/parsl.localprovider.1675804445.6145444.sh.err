2023/02/07 16:14:12.96 vine_worker[1177987] notice: stopping: abort signal received
