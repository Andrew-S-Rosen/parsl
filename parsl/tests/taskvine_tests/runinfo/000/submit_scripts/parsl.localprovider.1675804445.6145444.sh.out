Found cores : 4
Launching worker: 1
vine_worker: creating workspace /tmp/worker-239906-1177987
vine_worker: using 4 cores, 31742 MB memory, 74512 MB disk, 0 gpus
connected to manager cclws21.cse.nd.edu:50055 via local address 129.74.153.214:35804
disconnected from manager cclws21.cse.nd.edu:50055
vine_worker: deleting workspace /tmp/worker-239906-1177987
